script/src/main.rs

```diff
@@ -0,0 +1,81 @@
+use log::info;
+use sqlx::PgPool;
+use sqlx::Result;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+	dotenv::dotenv().expect("Unable to load environment variables from .env file");
+	env_logger::init(); // Initialize the logger
+
+	let db_url = std::env::var("DATABASE_URL").expect("Unable to read DATABASE_URL env var");
+	let debug_mode: bool = std::env::var("DEBUG_MODE").is_ok();
```

Comment:
IMO this is redundant to `RUST_LOG`. I would remove it completely and simply use `debug!`

Reply:
renamed to dry_mode